.rubocop.yml

```diff
@@ -1,12 +1,15 @@
-# Omakase Ruby styling for Rails
-inherit_gem: { rubocop-rails-omakase: rubocop.yml }
+inherit_gem:
+  rubocop-rails-omakase: rubocop.yml
+  
+Layout/IndentationWidth:
+  Enabled: true
 
-# Overwrite or add rules to create your own house style
-#
-# # Use `[a, [b, c]]` not `[ a, [ b, c ] ]`
-# Layout/SpaceInsideArrayLiteralBrackets:
-#   Enabled: false
-Layout/ElseAlignment:
-  Enabled: false
-Layout/EndAlignment:
-  Enabled: false
\ No newline at end of file
+Layout/IndentationStyle:
+  EnforcedStyle: spaces
+  IndentationWidth: 2
+
+Layout/IndentationConsistency:
+  Enabled: true
+
+Layout/SpaceInsidePercentLiteralDelimiters:
+  Enabled: true
```

Comment:
The Rails omakase config disables this one and prefers `%i[ items with space ]`.

That said, there is no Cop rule that can enforce this.  Given the OSS nature of this project, consistency is the most important thing; hence adding this rule in.